.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+<!---
+
+    Please initially open your PR as a draft until CI has passed and the
+    PR is ready for review. 
+
+    Once marked as ready for review, avoid force-pushing to the branch. 
+
+--->
+
+## PR type: 
+<!---   
+PR types: (check all applicable)
+        Breaking change: Any change that either drops support for a 
+        preview server version
+        Bug fix: Restores functionality of a currently broken feature
+        Build: Makes changes to how the image is built. Ex: update Vite 
+        version, changing the dockerfile
+        CI: Changes to our CI configuration files and scripts
+        Documentation: Updates in-code or external documentation
+        Feature: Introduce a new feature or rework an existing one
+        Optimization: Improves performance
+        Refactor: Improves code quality without changing the user experience
+--->
+
+- [ ] Breaking change
+- [ ] Bug fix 
+- [ ] Build
+- [ ] CI
+- [ ] Documentation
+- [ ] Feature
+- [ ] Optimization
+- [ ] Refactor
+- [ ] Other - Please specify
+
+## Changes:
+<!--- Provide the broad strokes of the changes made in a bulleted list --->
+
+-
+
+## Screenshots (for UI changes):
+<!--- Provide screenshots of all visual changes made in the PR --->
```

Comment:
Put this last so that it is not necessary to scroll through the images to get to the last text item.